### ISSUE TITLE
Could you please provide test VSR photos about PFNL?

### ISSUE THREAD

**Issue by wenchen4321** [psychopa4/PFNL#19]
Hello, Could you please provide the test VSR photos results about PFNL model? like Vid4 and UDM10.

**Comment by psychopa4** [comment]
> Hello, Could you please provide the test VSR photos results about PFNL model? like Vid4 and UDM10.

I suggest you try to use this code to generate the results.
If this is really inconvenient for you, please inform me again and I will upload the PFNL result in a few days.

**Comment by wenchen4321** [comment]
yes,I have learned tensorflow a little. It is difficult for me to run the code.I hope you can upload the test datasets. Thank you very much.

**Comment by psychopa4** [comment]
> yes,I have learned tensorflow a little. It is difficult for me to run the code.I hope you can upload the test datasets. Thank you very much.

My Google Drive runs out, thus I have to upload the PFNL result to [Baidu Drive](https://pan.baidu.com/s/14ca19-DiYeTxFWSX_tr4Lw).
Password：7zuh

**Comment by marwansalem** [comment]
if there is another source available as I cannot download from Baidu  without a Chinese account.
